docs/contribute/local-dev.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Local development setup"
+linkTitle: "Local development setup"
+landingSectionIndex: true
+weight: 100
+description: >
+  How to run a local greenhouse setup for development
+---
+
+# What is Greenhouse?
+
+Greenhouse is a [kubernetes operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) build with [kubebuilder](https://book.kubebuilder.io/introduction) and a UI on top of the k8s API.
+
+It expands the kubernetes API via CustomResourceDefinitions. The different aspects of the CRDs are reconciled by several [controllers](https://book.kubebuilder.io/cronjob-tutorial/controller-overview.html). It also acts as [an admission webhook](https://book.kubebuilder.io/reference/admission-webhook.html).
```

Comment:
Style: Proper nouns like Kubernetes, Greenhouse, Kubebuilder etc. should be capitalized.